frontend/src/bundles/auth/store/actions.ts

```diff
@@ -48,4 +48,13 @@ const loadCurrentUser = createAsyncThunk<
     return userApi.getCurrent();
 });
 
-export { loadCurrentUser, signIn, signUp };
+const logout = createAsyncThunk<Promise<void>, undefined, AsyncThunkConfig>(
+    `${sliceName}/logout`,
+    async (_, { extra }) => {
+        const { storage } = extra;
+
+        await storage.drop(StorageKey.TOKEN);
```

Comment:
```suggestion
   (_, { extra }) => {
        const { storage } = extra;

        return storage.drop(StorageKey.TOKEN);
```
There is no need for async await here

Reply:
Actually, async-await is needed. Take a look at how [BaseStorage](https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/blob/next/frontend/src/framework/storage/storage.package.ts) is implemented